extensions/amp-ima-video/validator-amp-ima-video.protoascii

```diff
@@ -67,3 +67,26 @@ tags: {  # <amp-ima-video>
     supported_layouts: RESPONSIVE
   }
 }
+
+tags: {
+  html_format: AMP
+  tag_name: "TRACK"
+  spec_name: "amp-ima-video > track"
+  mandatory_parent: "AMP-IMA-VIDEO"
+  # <amp-bind>
+  attrs: { name: "[label]" }
```

Comment:
Why is amp-bind enabled here rather than in the attr list? ("I don't want to risk breaking things" is a reasonable answer, but I was curious if there's a more interesting one.)

Reply:
I just copied existing rules for `amp-video`. actually realized we already had `subtitle` rule for `amp-ima-video` in main protoascii, was just missing the other rule which I added in another place.

Moving `[bind]` ones may make sense for all cases, but I would consider it outside of scope of this PR.

Reply:
Oh, I see, I guess variants of generic tags with a `mandatory_parent: "some-amp-extension"` actually show up in the main protoascii. (I wonder if we're consistent about this?)

Reply:
We attempt to group html tags into validator-main.protoascii and amp component tags within their amp component protoascii. However, some times if it's really specific that a generic html tag has some offshoot that only applies to a single amp component we may have it with the amp component. In either case where it's a generic html tag and there may be rules applying to it within an amp component protoascii we comment that there are rules in one or the other for those reading the protoascii to look up.